terraform-ecs-fargate-service/service.tf

```diff
@@ -0,0 +1,109 @@
+
+resource "aws_ecs_task_definition" "task_definition_api" {
+  family = var.service_name
+  execution_role_arn = var.task_execution_role_arn
+  task_role_arn = var.task_role_arn
+  requires_compatibilities = ["FARGATE"]
+  network_mode = "awsvpc"
+  cpu = 256
+  memory = 512
+  container_definitions = jsonencode([
+    {
+      "name": var.service_name,
+      "image": "${var.ecr_repository_url}:latest",
+      "portMappings": [
+            {
+                "containerPort": 4000
+            }
+        ],
+        "readonlyRootFilesystem": true,
+    },
+    { #checkov:skip=CKV_AWS_336: Ensure ECS containers are limited to read-only access to root filesystems - not possible for datadog-agent
+      "name": "datadog-agent",
+      "image": "datadog/agent:latest"
+    }
+  ])
```

Comment:
Jo tohle jsem tam přidal, protože to nešlo deploynout pokud ta služba ještě neexistuje. Tak jsem tam dal jen minimum aby to fungovalo.